plugins/win-mf/mf-plugin.cpp

```diff
@@ -0,0 +1,20 @@
+/* mf-plugin.cpp */
+#include <obs-module.h>
+#include <strsafe.h>
+#include <strmif.h>
+
+OBS_DECLARE_MODULE()
+OBS_MODULE_USE_DEFAULT_LOCALE(PLUGIN_NAME, "en-US")
+
+extern void RegisterMediaFoundationSource();
+
+bool obs_module_load(void)
+{
+	RegisterMediaFoundationSource();
+	return true;
+}
+
+void obs_module_unload()
+{
+	blog(LOG_INFO, "plugin unloaded");
```

Comment:
Either use a descriptive log message indicating which plugin unloaded, or omit the log message.

Reply:
Changed the it to "win-mf plugin unloaded". Is this good, or should I change is to something else?